outdatedbrowser/check.js

```diff
@@ -0,0 +1,21 @@
+function addLoadEvent(func) {
+  var oldonload = window.onload;
+  if (typeof window.onload != 'function') {
+    window.onload = func;
+  } else {
+    window.onload = function() {
+      if (oldonload) {
+        oldonload();
+      }
+      func();
+    };
+  }
+}
+addLoadEvent(function() {
+  outdatedBrowser({
```

Comment:
Add comment(s) describing why this function is here. It may not be obvious to someone reading the codebase for the first time.